resources/core/charts/console-backend-service/templates/cluster-role.yaml

```
@@ -14,6 +14,9 @@ rules:
   - apiGroups: [""]
     resources: ["pods","services", "configmaps", "secrets"]
     verbs: ["get", "list", "watch", "update", "delete"]
+  - apiGroups: [""]
+    resources: ["users"]
```

This will give permissions to console backend to perform group impersonation
```suggestion
    resources: ["users", "groups"]
```